### ISSUE TITLE
NameError: name '__version__' is not defined

### ISSUE THREAD

**Issue by 0xArcy** [takeshixx/python-ssllabs#13]
I am getting the following error while executing the program. 

```
Traceback (most recent call last):
  File "/home/arcy/python-ssllabs/ssllabs-cli.py", line 6, in <module>
    sys.exit(ssllabs.main())
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 369, in main
    cli_args = parse_arguments()
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 23, in parse_arguments
    version=__version__))
NameError: name __version__ is not defined
```

I've run this -> `python3 ssllabs-cli.py [mywebsiteurl]`

**Comment by takeshixx** [comment]
I pushed a quick change (8703a9751533317c462eb5d1b317558cff3589ce). Maybe depending on the Python version that failed, not sure. However, I cannot really test it atm because the API is down. (but help should work as well)

**Comment by 0xArcy** [comment]
Maybe. I am using the latest version to run this, so it could have been the new Python update. The API is still down though, I hope it gets back online.

**Comment by 0xArcy** [comment]
I tried running the API again and got this error which is not there in the documentation.

```
`ERROR:root:[API] unknown status code: 502, <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>502 Proxy Error</title>
</head><body>
<h1>Proxy Error</h1>
<p>The proxy server received an invalid
response from an upstream server.<br />
The proxy server could not handle the request<p>Reason: <strong>Error reading from remote server</strong></p></p>
<p>Additionally, a 502 Bad Gateway
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

ERROR:root:[API] unknown status code: 502, <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>502 Proxy Error</title>
</head><body>
<h1>Proxy Error</h1>
<p>The proxy server received an invalid
response from an upstream server.<br />
The proxy server could not handle the request<p>Reason: <strong>Error reading from remote server</strong></p></p>
<p>Additionally, a 502 Bad Gateway
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>
Traceback (most recent call last):
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 376, in main
    info = assessment.analyze(
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 280, in analyze
    if not self._check_api_info():
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 135, in _check_api_info
    response = self._handle_api_error(requests.get('{}info'.format(url))).json()
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 128, in _handle_api_error
    self._die_on_error('[API] unknown status code: {}, {}'.format(_status, error_message))
  File "/home/arcy/python-ssllabs/ssllabs/__init__.py", line 100, in _die_on_error
    raise AccessProblem(msg)
ssllabs.AccessProblem: [API] unknown status code: 502, <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>502 Proxy Error</title>
</head><body>
<h1>Proxy Error</h1>
<p>The proxy server received an invalid
response from an upstream server.<br />
The proxy server could not handle the request<p>Reason: <strong>Error reading from remote server</strong></p></p>
<p>Additionally, a 502 Bad Gateway
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>`

```

i have run -> `python3 ssllabs-cli.py example.com`

**Comment by takeshixx** [comment]
I don't think this is related to this project. Looks like you get an invalid response from the API (502 Bad Gateway).

**Comment by takeshixx** [comment]
Fixed in 60adfed2092701e3b3d790ea853975dd4dc53264.
